index.html

```diff
@@ -1 +1,61 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email
+experience for everyone. Recipients should be able to easily access the content of every email sent to them, regardless
+of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation
+and sending tools, and email clients to drive this vision forward at every step of the email journey.
+
+<p>Senders, recipients, and email clients share the common goals of improving user experience, accessibility, performance,
+consistency, and reliability. However, currently there is a lack of cooperation in trying to achieve these goals leading
+to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while
+delivering better results to the recipients.</p>
+
+<h2>How You Can Help</h2>
+
+<ol>
+  <li>PLACEHOLDER: Share on Twitter, LinkedIn, etc (links with share URLs?)</li>
+  <li>PLACEHOLDER: Join us (How? Link to form, GitHub repo)</li>
+</ol>
+
+<h2>Supporters</h2>
+
+<p>The Email Markup Consortium and its mission are supported by employees of Salesforce, Squarespace, Litmus, Email on Acid, Sparkpost, Constant Contact, ActionRocket, Parcel, and more.</p>
+
+<h2>Contributors</h2>
```

Comment:
For now this only includes those who were on yesterday’s call when we discussed it but we can put a call out to see which contributors would like to be listed (with optional links).

Reply:
I wonder if this should be a separate page, this could link to it? It comes with all sorts of connotations, how much do you need to do to become a 'contributor' - is it alphabetical by first/last name - when was the last time they contributed etc. Could also be used by companies to say - look we contributed 'x' is on the list and all they did was correct some spelling on a page. What does this add? 

Maybe - founding members plus link to contributors?

Reply:
I think the initial feeling is that having some people willing to put their names on the group both humanises it and maybe lends a bit of gravitas as we try to appeal to companies, etc.

At this point anyone who’s been involved in a meeting or comment thread to date could/should be listed. They’ve contributed. 

We did discuss in yesterday’s meeting how we might move forward with different ”membership levels.“ Self-identifying? Determined by a core team? etc. No answers yet though.

Reply:
I think keep it as is for now, then add additional content late on when we have more time and more content.